### PR TITLE
[BugFix] Fix cancel decommission statement miss analyze

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Review by Astralidea — DISMISSED** [review]

**Review by gengjun-git — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/sql/analyzer/Analyzer.java [hunk 1]
```diff
@@ -574,6 +569,21 @@ public Void visitPauseRoutineLoadStatement(PauseRoutineLoadStmt statement, Conne
...
+
+        @Override
+        public Void visitAlterSystemStatement(AlterSystemStmt statement, ConnectContext context) {
+            AlterSystemStmtAnalyzer.analyze(statement, context);
+            return null;
+        }
+
+        @Override
+        public Void visitCancelAlterSystemStatement(CancelAlterSystemStmt statement, ConnectContext context) {
+            AlterSystemStmtAnalyzer.analyze(statement, context);
```
Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/sql/analyzer/Analyzer.java:583]: Do we also need to check these two statments in PrivilegeCheckerV2?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 13 / 15 (86.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AlterSystemStmtAnalyzer.java|9|11|81.82%|[51, 52]|
|:large_blue_circle:|com/starrocks/sql/analyzer/Analyzer.java|4|4|100.00%|[]|

**Review by Astralidea — APPROVED** [review]

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by HangyuanLiu** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**